<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>embedding</class>
 <widget class="QWidget" name="embedding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout" name="trigger_word_container">
     <property name="spacing">
      <number>10</number>
     </property>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="enabled_checkbox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>enabledCheckbox</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="delete_button">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../../../gui/resources/feather.qrc">
       <normaloff>:/dark/icons/feather/dark/trash-2.svg</normaloff>:/dark/icons/feather/dark/trash-2.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="trigger_word_edit">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Some LoRA require a trigger word to activate.&lt;/p&gt;&lt;p&gt;Make a note here for your records.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Trigger words (comma separated)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../gui/resources/feather.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>trigger_word_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>embedding</receiver>
   <slot>action_changed_trigger_word(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_clicked_button_to_prompt()</slot>
  <slot>action_clicked_button_to_negative_prompt()</slot>
  <slot>action_toggled_embedding(bool)</slot>
  <slot>action_clicked_copy()</slot>
  <slot>action_changed_trigger_word(QString)</slot>
  <slot>action_clicked_button_deleted()</slot>
 </slots>
</ui>
